plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -138,6 +141,19 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       BATCH_SIZE,
     );
 
+    // Insert the refresh keys for the processed entity
+    await Promise.all(
+      options.refreshKeys.map(k => {
+        return tx<DbRefreshKeysRow>('refresh_keys')
+          .insert({
+            entity_ref: sourceEntityRef,
+            key: k.key,
+          })
+          .onConflict(['entity_ref', 'key'])
+          .ignore();
+      }),
+    );
```

Comment:
You can handle this the same way as the relations above, but even a bit simpler because we don't care about the previous values. So delete all entries for the given entity, then batch insert all the new ones. No need to handle conflicts since they won't happen with that strategy